bundle/edu.gemini.pot/src/main/scala/edu/gemini/pot/ModelConverters.scala

```diff
@@ -26,6 +26,25 @@ object ModelConverters {
     def toOldModel: skycalc.Angle = skycalc.Angle.degrees(angle.toDegrees)
   }
 
+  // We need a way to convert angles from [0,maxValOfUnit) to [-maxValOfUnit/2,maxValOfUnit/2) for a number of purposes.
+  private val maxRadians = 2 * math.Pi
+  private val maxDegrees = 360
+  private val maxArcmins = 60 * maxDegrees
+  private val maxArcsecs = 60 * maxArcmins
+  implicit class AngleNormalizer(val angle: Angle) extends AnyVal {
+    private def normalize(value: Double, maxValue: Double): Double = {
+      val half = maxValue / 2.0
+      val pos = value % maxValue
+      val neg = pos - maxValue
+      if (math.abs(pos) < math.abs(neg)) pos else neg
+    }
```

Comment:
`half` isn't used for anything and `pos` is always the same as `value` since they are being passed in from an `Angle.toDegrees` etc.?

We clearly need something like this put I'm not sure we want to add it as syntax or call it "normalized".  The [0,360) representation is arguably the normalized form.  Still, let's go with it temporarily until this sprint is over and we can plan for it.  At least this way we have a compile checked reminder of everywhere we have a problem.

Did you fix the offsetting issue with this?